Description: Upstream changes introduced in version 1.6-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 mari0 (1.6-1) unstable; urgency=low
 .
   * Initial version of package for Mari0 1.6.
 .
 The person named in the Author field signed this changelog entry.
Author: Mike Williamson <blessedbyvirtuosity@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ mari0-1.6/mari0
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+# TODO - respond to -h option
+love-unstable /usr/share/games/mari0/mari0_1.6.love
